recipes/nupic/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "nupic" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "048dae895ca505dddb46efb8ad062f58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - asteval ==0.9.1
+    - coverage ==3.7.1+    
+    - mock ==1.0.1
+    - ordereddict ==1.1
+    - psutil ==1.0.1
+    - pytest ==2.5.1
+    - pytest-cov ==1.6
+    - pytest-xdist ==1.8
+    - python-dateutil ==2.1
+    - pyyaml ==3.10
+    - unittest2 ==0.5.1
+    - validictory ==0.9.1
+    - pymysql ==0.6.2
+    - dbutils ==1.1
+    - pyproj ==1.9.3
+    - prettytable ==0.7.2
+    - nupic.bindings ==0.4.5
+    - numpy ==1.9.2
+
+  run:
+    - python
+    - asteval ==0.9.1
+    - coverage ==3.7.1
+    - mock ==1.0.1
+    - ordereddict ==1.1
+    - psutil ==1.0.1
+    - pytest ==2.5.1
+    - pytest-cov ==1.6
+    - pytest-xdist ==1.8
+    - python-dateutil ==2.1
+    - pyyaml ==3.10
+    - unittest2 ==0.5.1
+    - validictory ==0.9.1
```

Comment:
Ah, so @souravsingh you will need to write a conda recipe for `nupic.bindings` at https://github.com/numenta/nupic.core/tree/master/bindings/py before this will work. I am the release manager for that project so let me know how I can help on https://github.com/numenta/nupic/issues/3167.